[2021-03-01 21:01:04,935] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2020-09-04T00:00:00+00:00 [queued]>
[2021-03-01 21:01:04,939] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2020-09-04T00:00:00+00:00 [queued]>
[2021-03-01 21:01:04,939] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-03-01 21:01:04,939] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-03-01 21:01:04,939] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-03-01 21:01:04,949] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): processing_user> on 2020-09-04T00:00:00+00:00
[2021-03-01 21:01:04,954] {standard_task_runner.py:51} INFO - Started process 266042 to run task
[2021-03-01 21:01:04,959] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'processing_user', '2020-09-04T00:00:00+00:00', '--job-id', '1291', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp0qau1gp0']
[2021-03-01 21:01:04,960] {standard_task_runner.py:76} INFO - Job 1291: Subtask processing_user
[2021-03-01 21:01:04,999] {logging_mixin.py:103} INFO - Running <TaskInstance: user_processing.processing_user 2020-09-04T00:00:00+00:00 [running]> on host pop-os.localdomain
[2021-03-01 21:01:05,025] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=processing_user
AIRFLOW_CTX_EXECUTION_DATE=2020-09-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-09-04T00:00:00+00:00
[2021-03-01 21:01:05,034] {python.py:118} INFO - Done. Returned value was: None
[2021-03-01 21:01:05,036] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=processing_user, execution_date=20200904T000000, start_date=20210302T020104, end_date=20210302T020105
[2021-03-01 21:01:05,055] {taskinstance.py:1195} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-01 21:01:05,089] {local_task_job.py:118} INFO - Task exited with return code 0
